stdlib/2.7/sys.pyi

```diff
@@ -86,6 +86,9 @@ path_hooks = ...  # type: List[Any]
 path_importer_cache = ...  # type: Dict[str, Any]
 displayhook = ...  # type: Optional[Callable[[int], None]]
 excepthook = ...  # type: Optional[Callable[[type, BaseException, TracebackType], None]]
+exc_type = ...  # type: type
+exc_value = ...  # type: BaseException
```

Comment:
I wonder if we can ignore that detail? Does pytype understand the difference between old-style and new-style classes? Mypy doesn't -- it just assumes all classes have type `type` and insists that all exception classes inherit from BaseException.

If we go with the mypy model, all three values should just be declared `Optional[...]`. If we go with Matthias' suggestion then I'm not sure what to use for exc_type -- maybe `Union[None, type, types.ClassType]`. In either case `exc_traceback` should be Optional.

(Currently mypy doesn't care about the lack of Optional, but it will.)